application/templates/cms_autosave/_methodology.html

```diff
@@ -17,7 +17,7 @@ <h3 class="heading-small">Rounding</h3>
         <h3 class="heading-small">Related publications</h3>
         {{ editable_textarea(form.related_publications, empty_message="Click to add", tab_index=11) }}
 
-        {{ editable_link(link_url_field=form.qmi_url, link_text_field=None, empty_message="Click to add", tab_index=12) }}
+        {{ editable_link(link_url_field=form.qmi_url, link_text_field=None, empty_message="Click to add Quality and methodology information link", tab_index=12) }}
```

Comment:
It now has this message on the page whether or not there is a value set for the URL.